yun/libraries/Parse/src/internal/ParseClient.h

```diff
@@ -35,6 +35,9 @@
 
 /*! \class ParseClient
  *  \brief Class responsible for Parse connection
+ *
+ *  NOTE: A global ParseClient object with name "Parse" is defined in SDK,
```

Comment:
...object with **the** name Parse is defined in **the** SDK...

Add the two "the"